wp-content/themes/csisnuclearnetwork/assets/_scss/base/_typography.scss

```diff
@@ -16,3 +22,24 @@ a {
     outline: 0;
   }
 }
+
+/*----------  Utility Classes  ----------*/
+.text--bold {
+  font-weight: 700;
+}
+
+.text--italic {
+  font-style: italic;
+}
+
+.text--caps {
+  text-transform: uppercase;
+}
+
+.text--short {
+  @extend %text-style-paragraph-short;
```

Comment:
It might be better to rename this placeholder so it doesn't have `paragraph` in it, so if you end up needing to reference it elsewhere that isn't a "paragraph" the naming isn't confusing.